examples/XML/maps/ChaoticThreshold.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<Map xmlns="http://www.hybrasyl.com/XML/Maps" Id="300" X="30" Y="30">
+  <Name>Chaotic Threshold</Name>
+  <Npcs>
+    <Npc Name="Nylarlathotep" X="12" Y="6" Direction="0"/>
```

Comment:
Nylarlathotep xml missing. Results in KeyNotFoundException on server start.

Reply:
Fixed